python_benchmarks/test_softmax_fwd.py

```diff
@@ -48,6 +49,11 @@ def softmax_fwd_fn(inputs: list):  # [in_tensor, reduction_axis]
     return torch.nn.functional.softmax(inputs[0], inputs[1])
 
 
+def softmax_fwd_iobytes(size: tuple, dtype: torch.dtype):
+    # Total IO bytes = input + output
+    return int(np.prod(size) * dtype.itemsize * 2)
```

Comment:
These functions will be added to the benchmarks to manually compute IO bytes to avoid mismatch in IOBytes since some intermediate outputs will not be returned with torch but are returned in nvfuser definitions.